src/parse-nm.c

```diff
@@ -697,6 +699,14 @@ netplan_parser_load_keyfile(NetplanParser* npp, const char* filename, GError** e
         }
     }
 
+    /* Handle VRFs */
+    if (nd_type == NETPLAN_DEF_TYPE_VRF) {
+        if (g_key_file_has_key(kf, "vrf", "table", NULL)) {
+            nd->vrf_table = g_key_file_get_integer(kf, "vrf", "table", NULL);
```

Comment:
todo: According to https://developer-old.gnome.org/NetworkManager/stable/settings-vrf.html the keyfile's `vrf.table` field is of type `uint32`, so we should probably use something like this, to avoid truncation:

```suggestion
            nd->vrf_table = g_key_file_get_uint64(kf, "vrf", "table", NULL);
```